e2e/test/scenarios/visualizations-tabular/smartscalar-trend.cy.spec.js

```diff
@@ -1,15 +1,194 @@
-import { restore } from "e2e/support/helpers";
+import Color from "color";
+import { colors } from "metabase/lib/colors";
+import { menu, popover, restore } from "e2e/support/helpers";
 import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
 
 const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
 
-describe("scenarios > visualizations > scalar", () => {
+describe("scenarios > visualizations > trend chart (SmartScalar)", () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
   });
 
-  it("trend visualization should work regardless of column order (metabase#13710)", () => {
+  it("should allow data settings to be changed and display should reflect changes", () => {
+    cy.createQuestion(
+      {
+        name: "13710",
+        query: {
+          "source-table": ORDERS_ID,
+          aggregation: [["count"], ["sum", ["field", ORDERS.TOTAL, null]]],
+          breakout: [
+            ["field", ORDERS.CREATED_AT, { "temporal-unit": "month" }],
+          ],
+        },
+        display: "smartscalar",
+      },
+      { visitQuestion: true },
+    );
+
+    cy.findByTestId("viz-settings-button").click();
+    cy.findByTestId("chartsettings-sidebar").findByText("Data").click();
+
+    // primary number
+    cy.findByTestId("scalar-container").findByText("344");
+    cy.findByTestId("chartsettings-sidebar").findByText("Count").click();
+    popover().within(() => {
+      // should only have two options
+      cy.get('[role="option"]').should("have.length", 2);
+
+      // selected should be highlighted
+      cy.get('[aria-label="Count"]').should(
+        "have.css",
+        "background-color",
+        Color(colors.brand).string(),
```

Comment:
E2E specs are not the best option to check styles, I'd suggest to avoid adding such assertions. Instead we could query an element with `aria-selected="true"` and verify it is "Count"